### PR TITLE
First pass at l10n on add-on

### PR DESCRIPTION
**Pull request by lmorchard**
Issue #557

I read through all the JS modules in the add-on, and I think I found all the strings. Might have missed some, though.

My goal was just to see if I could get these changes working with the standard `sdk/l10n` approach in [the Localization tutorial](https://developer.mozilla.org/en-US/Add-ons/SDK/Tutorials/l10n). But, this seems to be a requirement:

> although note that the .properties files need to be in /locale/%LOCALE%/whatever.properties 
> instead of what min vid is doing.

So, this might need a follow up / more work to get things working in Pontoon?

### CODE REVIEW

**Comment by lmorchard** [comment]
Hmm, failed an add-on test in circleci that passed locally. This could still use some general review, though. I'll poke and see why that failed.

File: addon/locale/en-US.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# LOCALIZER NOTE: Placeholder is experiment title
+survey_show_rating_label = Please rate %s
+# LOCALIZER NOTE: Placeholder is experiment title
+survey_rating_thank_you = Thank you for rating %s.
+survey_rating_survey_button = Take a Quick Survey
+# LOCALIZER NOTE: Placeholder is experiment title
+survey_launch_survey_label = The %s experiment has ended. What did you think?
+
+notification_via = via testpilot.firefox.com
```
Comment (pdehaan) [inline addon/locale/en-US.properties]: Should this URL be a placeholder to prevent localizers from potentially changing it?

**Comment by lmorchard** [comment]
Think I fixed the broken test and placeholder-ized the host name in the notification message.

**Comment by lmorchard** [comment]
Grr, lint.

**Comment by chuckharmston** [comment]
I don't have any substantive feedback; code looks good, seems to work well.

**Review by chuckharmston — APPROVED** [review]

**Comment by lmorchard** [comment]
Going to merge this and follow up on [making the path Pontoon-friendly](https://github.com/mozilla/testpilot/issues/557) in another PR today.

**Pull request merged**